template/.github/workflows/shipwright-test.yml

```
@@ -0,0 +1,23 @@
+# This is a basic workflow to help you get started with Actions
+
+name: Shipwright Tokens Example
+
+# Controls when the workflow will run
+on:
+  # Triggers the workflow on push or pull request events but only for the "main" branch
+  pull_request:
+    branches: ["main"]
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  # This workflow contains a single job called "build"
+  build:
+    runs-on: ubuntu-latest
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+      - name: Shipwright Tokens
+        uses: headwayio/shipwright-tokens@main
+        with:
+          styleSystem: mui
+          outputFolder: styles
```

Do you think there should be a comment here with the different options? Maybe something like:
```suggestion
        # Pass in options to the Github Action: styleSystem, outputFolder, etc.
        with:
          # Options: mui, scss, tailwind
          styleSystem: mui
          # This is relative to the root of the repo
          outputFolder: styles
```

---

That's great! I'll commit these changes. 

I think we should also have more detailed documentation about what each option does and what needs to be setup in the main projects' README